Unicore fails compilation on TS 3.6

```
node_modules/unicore/dist/lib/fileUploader/gcpBucket.d.ts:1:23 - error TS2688: Cannot find type definition file for 'multer'.

1 /// <reference types="multer" />
                        ~~~~~~

node_modules/unicore/dist/lib/fileUploader/gcpBucket.d.ts:12:29 - error TS2694: Namespace 'Express' has no exported member 'Multer'.

12 declare type File = Express.Multer.File | {
                               ~~~~~~

node_modules/unicore/dist/lib/server.d.ts:2:23 - error TS2688: Cannot find type definition file for 'server-destroy'.

2 /// <reference types="server-destroy" />
                        ~~~~~~~~~~~~~~


Found 3 errors.

```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- add missing exported deps to dev (fix #1)
+
+## [0.1.8] - 2019-09-19
+### Added
+- expose file upload
 
 ## [0.1.7] - 2019-05-17
 ### Fixed
@@ -19,7 +25,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Added base implementation
 
-[Unreleased]: https://github.com/AckeeCZ/unicore/compare/v0.1.7...HEAD
+[Unreleased]: https://github.com/AckeeCZ/unicore/compare/v0.1.8...HEAD
+[0.1.8]: https://github.com/AckeeCZ/unicore/compare/v0.1.7...v0.1.8
 [0.1.7]: https://github.com/AckeeCZ/unicore/compare/v0.1.6...v0.1.7
 [0.1.6]: https://github.com/AckeeCZ/unicore/compare/v0.1.4...v0.1.6
 [0.1.4]: https://github.com/AckeeCZ/unicore/compare/298eccb...v0.1.4

--- a/package.json
+++ b/package.json
@@ -30,6 +30,8 @@
     "@google-cloud/storage": "^2.5.0",
     "@types/cors": "^2.8.5",
     "@types/raven": "^2.5.3",
+    "@types/multer": "^1.3.7",
+    "@types/server-destroy": "^1.0.0",
     "body-parser": "^1.18.3",
     "cors": "^2.8.5",
     "express": "^4.16.4",
@@ -40,9 +42,7 @@
   "devDependencies": {
     "@types/express": "^4.16.1",
     "@types/jest": "^24.0.11",
-    "@types/multer": "^1.3.7",
     "@types/request-promise": "^4.1.43",
-    "@types/server-destroy": "^1.0.0",
     "@types/supertest": "^2.0.7",
     "cross-env": "^5.2.0",
     "husky": "^1.3.1",

